gammapy/maps/region.py

```diff
@@ -415,3 +415,29 @@ def union(self, other):
                 self._region = self.region.union(other.region)
             else:
                 self._region = other.region
+
+    def plot_region(self, ax=None, **kwargs):
+        """Plot region in the sky.
+
+        Parameters
+        ----------
+        ax : `~astropy.vizualisation.WCSAxes`
+            Axes to plot on. If no axes are given,
+            the region is shown using the minimal
+            equivalent WCS geometry.
+        **kwargs : dict
+            Keyword arguments forwarded to `~regions.PixelRegion.as_artist`
+        """
+        import matplotlib.pyplot as plt
+        from matplotlib.collections import PatchCollection
+
+        if ax is None:
```

Comment:
We could still keep the previous behaviour, with `ax = plt.gca()`, which works if the "current" axis is a `WCSAxes` and just catch the case, where the current axis is not a was one...

Reply:
you're right, I changed it, thanks!